src/plots/cartesian/axis_defaults.js

```diff
@@ -75,6 +75,12 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
     coerce('range');
     containerOut.cleanRange();
 
+    coerce2('showspikes');
```

Comment:
Change made and tested locally. Will hold off on commit to see if there's any more feedback.